tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "compilerOptions": {
     "resolveJsonModule": true,
     /* Basic Options */
-    "target": "ES3",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
+    "target": "ES5",                          /* Specify ECMAScript target version: 'ES5' (default), 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */

```

Comment:
`Target ES3` is deprecated so I've upgraded to `ES5`. This also allows us to use newer array methods like `includes`.